src/services/completions.ts

```diff
@@ -4868,16 +4868,17 @@ function tryGetObjectTypeDeclarationCompletionContainer(sourceFile: SourceFile,
         case SyntaxKind.CommaToken: // class c {getValue(): number, | }
             return tryCast(contextToken.parent, isObjectTypeDeclaration);
         default:
-            if (!isFromObjectTypeDeclaration(contextToken)) {
-                // class c extends React.Component { a: () => 1\n| }
-                if (getLineAndCharacterOfPosition(sourceFile, contextToken.getEnd()).line !== getLineAndCharacterOfPosition(sourceFile, position).line && isObjectTypeDeclaration(location)) {
+            if (isObjectTypeDeclaration(location)) {
+                // class C extends React.Component { a: () => 1\n| }
+                // class C { prop = ""\n | }
+                if (isClassLike(location) && getLineAndCharacterOfPosition(sourceFile, contextToken.getEnd()).line !== getLineAndCharacterOfPosition(sourceFile, position).line) {

```

Comment:
Question: why do we need the `isClassLike(location)` check here?

Reply:
The context of the issue is related to properties declaration, however, I'm not sure if it is valid for `InterfaceDeclaration`, `TypeLiteralNode` contexts.

Reply:
I think we can get away with removing the check. The function doesn't seem to be restricted to classes. I'm not sure removing the check will have an immediate effect anyway, since I tried it around for a bit and we don't seem to offer completions inside interfaces and type literals, but removing it might spare us some future confusion.